handler.go

```diff
@@ -512,7 +525,7 @@ func (h *Handler) serveDataNodes(w http.ResponseWriter, r *http.Request, u *User
 }
 
 // serveCreateDataNode creates a new data node in the cluster.
-func (h *Handler) serveCreateDataNode(w http.ResponseWriter, r *http.Request, u *User) {
+func (h *Handler) serveCreateDataNode(w http.ResponseWriter, r *http.Request, _ *User) {
```

Comment:
That's cool. I was bugged by the fact that I couldn't use `u` in the function body.